docker-compose.yml

```diff
@@ -3,37 +3,54 @@ version: "3.7"
 services:
   postgres:
     image: postgres:15
-    expose:
-      - 5432
-    environment:
-      - "POSTGRES_USER=ayon"
-      - "POSTGRES_PASSWORD=ayon"
-      - "POSTGRES_DB=ayon"
-    volumes:
-      - "/etc/localtime:/etc/localtime:ro"
-      - "db:/var/lib/postgresql/data"
     restart: unless-stopped
     healthcheck:
       test: ["CMD-SHELL", "pg_isready -U ayon"]
       interval: 5s
       timeout: 5s
       retries: 5
 
+    expose: [5432]
+
+    volumes:
+      - "/etc/localtime:/etc/localtime:ro"
+      - "db:/var/lib/postgresql/data"
+
+    environment:
+      - "POSTGRES_USER=ayon"
+      - "POSTGRES_PASSWORD=ayon"
```

Comment:
At some point we should be really enforcing people to change this to something else...